pkg/storage/system_intake_test.go

```diff
@@ -489,7 +462,11 @@ func (s *StoreTestSuite) TestFetchSystemIntakesByEuaID() {
 		intake := testhelpers.NewSystemIntake()
 		intake2 := testhelpers.NewSystemIntake()
 		intake2.EUAUserID = intake.EUAUserID
-		intake2.Status = models.SystemIntakeStatusWITHDRAWN
+
+		// set archived at for intake2
+		now := time.Now()
+		intake2.ArchivedAt = &now
```

Comment:
double check

Reply:
Makes sense. But maybe cleaner to use this, now that we have it!
```suggestion
		intake2.ArchivedAt = helpers.PointerTo(time.Now())
```